### DISCUSSION TITLE
Connect with jackett running through gluetun... How?

### DISCUSSION THREAD

**Discussion by truthsword** [FlareSolverr/FlareSolverr#1066] [Q&A] [answered]
flaresolverr connects well with jackett, but I must run jackett through gluetun (vpn container). When I do so, flaresolverr won't connect. 

How do I get this to work? Must I place flaresolverr under the same glutetun container? Or? 
Thank you.

**Comment by ilike2burnthing** [comment, marked as answer]
> Must I place flaresolverr under the same glutetun container?

Yes.

**Reply by truthsword** [reply]
Thank you. Worked as noted.
